katcp/test/test_core.py

```diff
@@ -7,8 +7,12 @@
 """Tests for the katcp utilities module.
    """
 
+# Python 2/3 compatibility stuff
+from builtins import str
+from builtins import object
 import logging
-import unittest2 as unittest
+import unittest
```

Comment:
unittest2 is  a backport of Python 2.7's unittest to older versions. AFAIK the only difference between python 2.7+ unittest and unittest2 are some of the message printing defaults, but nothing substantial.